### PR TITLE
Windows Defender Threats

### PR DESCRIPTION
**Pull request by b0park**
Pulls a computers past or active Windows Defender Threat history. Useful in generating reports and alerts for additional mitigation or follow up.

### CODE REVIEW

File: PowerShell Scanners/Windows Defender Threats/Windows Defender Threat Detection.ps1 [hunk 1]
```diff
@@ -0,0 +1,11 @@
+﻿if (-Not(Get-Command Get-MpComputerStatus -ErrorAction SilentlyContinue)) {
+    throw "Unable to find Get-MpComputerStatus. Available on Windows 10/Server 2016 or higher"
```
Comment (Colby-PDQ) [inline PowerShell Scanners/Windows Defender Threats/Windows Defender Threat Detection.ps1]: Get-MpComputerStatus needs to be changed to Get-MpThreatDetection.

File: PowerShell Scanners/Windows Defender Threats/PS - Windows Defender Threats.xml [hunk 2]
```diff
@@ -0,0 +1,25 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (Colby-PDQ) [inline PowerShell Scanners/Windows Defender Threats/PS - Windows Defender Threats.xml]: Please rename this file to `Scan Profile.xml`: https://github.com/pdq/PowerShell-Scanners/blob/master/Policies/Contributing.md#folder-structure

**Review by Colby-PDQ — CHANGES_REQUESTED** [review]

File: PowerShell Scanners/Windows Defender Threats/Windows Defender Threat Detection.ps1 (hunk 1 above)
Comment (b0park) [reply PowerShell Scanners/Windows Defender Threats/Windows Defender Threat Detection.ps1]: Copy paste failure fixed.

File: PowerShell Scanners/Windows Defender Threats/PS - Windows Defender Threats.xml (hunk 2 above)
Comment (b0park) [reply PowerShell Scanners/Windows Defender Threats/PS - Windows Defender Threats.xml]: I've renamed according to contributing guidelines, I'm new to Git though, should I do a new pull request?

Comment (steviecoaster) [reply PowerShell Scanners/Windows Defender Threats/PS - Windows Defender Threats.xml]: Nope. Additional commits made to the branch will get added to this PR., and it looks like they have accoding to de06d98

**Review by Colby-PDQ — APPROVED** [review]
Looks great! Thank you for contributing to this project!

**Pull request merged**